flepimop/gempyor_pkg/src/gempyor/parameters.py

```diff
@@ -123,30 +123,25 @@ def __init__(
                     print("loaded col :", sorted(list(df.columns)))
                     print("geodata col:", sorted(subpop_names))
                     raise ValueError(
-                        f"""ERROR loading file {fn_name} for parameter {pn}: the number of non 'date'
-                    columns are {len(df.columns)}, expected {len(subpop_names)} (the number of subpops) or one."""
+                        f"Issue loading file '{fn_name}' for parameter '{pn}': "
+                        f"the number of non-'date' columns is '{len(df.columns)}', "
+                        f"expected '{len(subpop_names)}' (number of subpopulations) or one."
                     )
 
                 df = df[str(ti) : str(tf)]
                 if not (len(df.index) == len(pd.date_range(ti, tf))):
                     print("config dates:", pd.date_range(ti, tf))
                     print("loaded dates:", df.index)
                     raise ValueError(
-                        f"""ERROR loading file {fn_name} for parameter {pn}: 
-                    the 'date' entries of the provided file do not include all the days specified to be modeled by 
-                    the config. the provided file includes {len(df.index)} days between {str(df.index[0])} to {str(df.index[-1])}, 
-                    while there are {len(pd.date_range(ti, tf))} days in the config time span of {ti}->{tf}. The file must contain entries for the
-                    the exact start and end dates from the config. """
+                        f"Issue loading file '{fn_name}' for parameter '{pn}': " 
+                        f"The 'date' entries of the provided file do not include all the days specified to be modeled by the config."
```

Comment:
I think the dates were pretty handy to have in the message, just didn't need them multiple times or the number of days (users can get that from the dates). Maybe something along the lines of:
```
f"The file '{fn_name}' for parameter '{pn}' spans {str(df.index[0])} to {str(df.index[-1])}, but the config spans {ti} to {tf}."
```
Or similar, plus formatting.